server/routes/users.js

```diff
@@ -82,4 +84,43 @@ router.get('/:name/productslist/:id', async function (req, res) {
 	}
 });
 
+/*router.get('/:id', verifyToken, async (req, res) => {
+	try {
+		const user = await User.findById(req.params.id);
+		res.status(200).send(user);
+	} catch (err) {
+		console.error('User not found');
+		res.status(500);
+	}
+});*/
```

Comment:
can we remove these

Reply:
Extremely sorry for that! I forgot to clean the code! Will remember!